sympy/series/tests/test_limits.py

```diff
@@ -544,8 +544,8 @@ def test_order_oo():
 
 
 def test_issue_5436():
-    raises(NotImplementedError, lambda: limit(exp(x*y), x, oo))
-    raises(NotImplementedError, lambda: limit(exp(-x*y), x, oo))
+    assert limit(exp(x*y), x, oo) == Limit(exp(x*y), x, oo, dir='-')
+    assert limit(exp(-x*y), x, oo) == Limit(exp(-x*y), x, oo, dir='-')
```

Comment:
Naah, I don't think we should be doing this !

This is what we have currently 
```
>>> limit(exp(x*y), x, oo)
NotImplementedError: Result depends on the sign of sign(y)
```

which I think is what we want. I mean this is a place where we can prompt the user to give us more detial and then we can compute the limit for the user.
```
>>> y = Symbol("y", positive=True)
>>> limit(exp(x*y), x, oo)
oo
```